en/docs/deploy-with-wso2-choreo.md

```diff
@@ -0,0 +1,7 @@
+# Deploy with WSO2 Choreo
+
+You can manage your Ballerina projects by deploying them in [WSO2 Choreo](https://wso2.com/choreo/). The combination of the Ballerina VS Code extension and [Choreo VS Code extension](https://marketplace.visualstudio.com/items?itemName=WSO2.choreo) enables this functionality. 
```

Comment:
- is it correct to say `manage .... by deploying`? What is meant by manage?
- is it correct to say projects?

Reply:
Changed to `You can deploy the Ballerina projects you create in [WSO2 Choreo](https://wso2.com/choreo/). `.